### ISSUE TITLE
Two people video for a while，"Unfortunately, something went wrong."

### ISSUE THREAD

**Issue by yangyingdong** [jitsi/docker-jitsi-meet#728]
I can create room very well but after starting conversation with aother parties. I have got this message "Unfortunately, something went wrong."About once in 20 seconds.thanks in advanced

**Comment by yangyingdong** [comment]
[192.168.1.239-1598922679704.log](https://github.com/jitsi/docker-jitsi-meet/files/5153368/192.168.1.239-1598922679704.log)
this is my console log.

**Comment by alexitosrv** [comment]
Yeah, I have observed the same with the latest release on CentOS 7 and using docker-compose.

**Comment by yangyingdong** [comment]
> Yeah, I have observed the same with the latest release on CentOS 7 and using docker-compose.
But I don't know how to solve it.

**Comment by alexitosrv** [comment]
If someone is interested, or if at least can provide pointers, this is the screenshot ![Screenshot (268)](https://user-images.githubusercontent.com/3821436/92290041-5551eb00-eed8-11ea-9e5c-54f9161cb882.png) of the part where the connection is lost for both parties at the same time.

I was looking into potential timeouts hardcoded, but the only fragment of code that it's also close to the 20 seconds cut out is this call of handleRayoIQ((RayoIqProvider.DialIq) iqRequest, 2, null); at line 125 in MeetExtensionsHandler.java, although that is 15 seconds, but it's very strange. For the connection to drop at the same time for both parties, the reason probably it's at the server side, not sure if it's in the web app. The connection doesn't drop at all when it's only one user sharing his feed.

I also reverted to the code of 3 previous releases and the problem happened as well, so probably it has to do with something else. 

I also disabled selinux, just because, but that doesn't have anything to do with it.

**Comment by wangqi060934** [comment]
I have the same problem. I captured the console log from chrome:
![image](https://user-images.githubusercontent.com/1433283/94161118-ec96d680-feb7-11ea-85a8-29c2df1149d0.png)

**Comment by wangqi060934** [comment]
this log shows a js error:
![image](https://user-images.githubusercontent.com/1433283/94163278-3b457000-feba-11ea-926f-05870711b3ad.png)

**Comment by wangqi060934** [comment]
@yangyingdong @alexitosrv I have solved my problem.I install jitsi-meet by docker in LAN, but I don't set DOCKER_HOST_ADDRESS in .env. Check the doc:https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker#running-behind-nat-or-on-a-lan-environment
